<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="6">
     <widget class="QLineEdit" name="lineEdit_display">
      <property name="font">
       <font>
        <family>DejaVu Math TeX Gyre</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="button1">
      <property name="font">
       <font>
        <family>DejaVu Math TeX Gyre</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="button2">
      <property name="font">
       <font>
        <family>DejaVu Math TeX Gyre</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="button3">
      <property name="font">
       <font>
        <family>DejaVu Math TeX Gyre</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="buttonAdd">
      <property name="font">
       <font>
        <family>DejaVu Math TeX Gyre</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="buttonAbs">
      <property name="font">
       <font>
        <family>DejaVu Math TeX Gyre</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>|x|</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5" rowspan="4">
     <widget class="QPushButton" name="buttonEquals">
      <property name="font">
       <font>
        <family>DejaVu Math TeX Gyre</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="button4">
      <property name="font">
       <font>
        <family>DejaVu Math TeX Gyre</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="button5">
      <property name="font">
       <font>
        <family>DejaVu Math TeX Gyre</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="button6">
      <property name="font">
       <font>
        <family>DejaVu Math TeX Gyre</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="buttonSubtract">
      <property name="font">
       <font>
        <family>DejaVu Math TeX Gyre</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="buttonSin">
      <property name="font">
       <font>
        <family>DejaVu Math TeX Gyre</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>sin(x)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="button7">
      <property name="font">
       <font>
        <family>DejaVu Math TeX Gyre</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="button8">
      <property name="font">
       <font>
        <family>DejaVu Math TeX Gyre</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="button9">
      <property name="font">
       <font>
        <family>DejaVu Math TeX Gyre</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="buttonMultiply">
      <property name="font">
       <font>
        <family>DejaVu Math TeX Gyre</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="buttonLog">
      <property name="font">
       <font>
        <family>DejaVu Math TeX Gyre</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>log(n)(x)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="button0">
      <property name="font">
       <font>
        <family>DejaVu Math TeX Gyre</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="buttonClear">
      <property name="font">
       <font>
        <family>DejaVu Math TeX Gyre</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="buttonDivide">
      <property name="font">
       <font>
        <family>DejaVu Math TeX Gyre</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="buttondB">
      <property name="font">
       <font>
        <family>DejaVu Math TeX Gyre</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>dB(x)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>615</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
